### DISCUSSION TITLE
Is there any way to set the cursor position?

### DISCUSSION THREAD

**Discussion by camillodev** [ueberdosis/tiptap#1383] [General Questions] [answered]
Hi guys!

At the mounted Vue JS lifecycle, I'm creating the tip tap editor, and I need to at this moment set the cursor position and the focus to the position number I'm getting from the props. 
Can Anyone help me? I have no idea how to make it. 

Is there any way to set the cursor position? I only founded out how to get the current cursor position. 

I'm using Tiptap V2

**Comment by akirarika** [comment]
I also want to know

**Comment by hanspagel** [comment, marked as answer]
You can use the `setTextSelection` command to set the cursor.

```js
editor.chain().focus().setTextSelection(10).run()
```

Does that help?

**Reply by akirarika** [reply]
if I want to move to the end of the editor, what should I do?

**Reply by hanspagel** [reply]
Try `editor.commands.focus('end')`! ✌️

**Reply by camillodev** [reply]
I tried and got an error: setTextSelection is not a function.
I'm sorry I forgot to mention that I'm using Tiptap v2.

**Reply by philippkuehn** [reply]
> I tried and got an error: setTextSelection is not a function.
> 
> I'm sorry I forgot to mention that I'm using Tiptap v2.

try updating your dependencies.

**Reply by camillodev** [reply]
It Worked! Thanks So much!!!!
